controllers/votes.controllers.js

```diff
@@ -0,0 +1,24 @@
+const { updateArticleVotes } = require("../models/votes.models");
+
+exports.patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const { inc_votes } = request.body;
+
+  if (isNaN(article_id)
+) {
```

Comment:
Nice way of handling this - if you removed this logic and used a NaN value to request data from the database i believe you'd receive a PSQL error - how might you choose to handle that scenario?